cpp/utils.cpp

```diff
@@ -62,7 +67,7 @@ JSVariant toVariant(jsi::Runtime &rt, const jsi::Value &value) {
     auto obj = value.asObject(rt);
 
     if (!obj.isArrayBuffer(rt)) {
-      throw std::invalid_argument(
+      throw std::runtime_error(
```

Comment:
Let's only use `std::runtime_error` there are some problems with RN and Android where exceptions are not matching the parent std::exception class